tests/app/dao/test_services_dao.py

```diff
@@ -1099,3 +1071,79 @@ def create_email_sms_letter_template():
     template_two = create_template(service=service, template_name='2', template_type='sms')
     template_three = create_template(service=service, template_name='3', template_type='letter')
     return template_one, template_three, template_two
+
+
+@freeze_time("2019-12-02 12:00:00.000000")
+def test_dao_find_services_sending_to_tv_numbers(notify_db_session, fake_uuid):
+    service_1 = create_service(service_name="Service 1", service_id=fake_uuid)
```

Comment:
Do we need to use fake_uuid?

Reply:
We need an uuid, as we assert on it below, to check the result we get is for the right service.

Reply:
You can just use the id from service_1.id